tests/suite/test_app_protect_integration.py

```diff
@@ -382,6 +387,46 @@ def test_ap_sec_logs_on(
         assert f'request_status="passed"' in log_contents
         assert f'outcome="PASSED"' in log_contents
 
+    @pytest.mark.startup
+    def test_ap_pod_startup(
+        self,
+        request,
+        kube_apis,
+        ingress_controller_prerequisites,
+        crd_ingress_controller_with_ap,
+        appprotect_setup,
+        test_namespace,
+    ):
+        """
+        Log pod startup time while scaling up from 0 to 1
+        """
+        src_syslog_yaml = f"{TEST_DATA}/appprotect/syslog.yaml"
+        create_items_from_yaml(kube_apis, src_syslog_yaml, test_namespace)
+
+        syslog_ep = get_service_endpoint(kube_apis, "syslog-svc", test_namespace)
+
+        # items[-1] because syslog pod is last one to spin-up
+        syslog_pod = kube_apis.v1.list_namespaced_pod(test_namespace).items[-1].metadata.name
```

Comment:
Perhaps it would be safer to look at pod name or labels rather then relying on the pod having a certain index?